pkg/transfer/local/pull.go

```diff
@@ -241,3 +248,20 @@ func fetchHandler(ingester content.Ingester, fetcher remotes.Fetcher, pt *Progre
 		}
 	}
 }
+
+// getSupportedPlatform returns a matched platform comparing input UnpackConfiguration to the supported platform/snapshotter combinations
+// If input platform didn't specify snapshotter, default will be used if there is a match on platform.
+func getSupportedPlatform(uc image.UnpackConfiguration, supportedPlatforms []unpack.Platform) (bool, unpack.Platform) {
```

Comment:
Think this function is worth unit testing, in this PR or follow up is fine

Reply:
I has that on my Untracked files list.   Finalizing it.